app/models/column_preference.rb

```diff
@@ -15,10 +15,11 @@ class ColumnPreference < ApplicationRecord
   validates :column_name, presence: true, inclusion: { in: proc { |p|
                                                              valid_columns_for(p.action_name)
                                                            } }
+  scope :products, -> { where(action_name: 'products_v3_index') }
```

Comment:
I think we should rename this to `bulk_edit_product` so it's clear we are talking about a page and not a product.

Reply:
It's fixed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12854/commits/5be53a40a97fcf156caf920553bf0df6638cde35